README.md

```diff
@@ -271,27 +270,27 @@ But we don't just want to blindly do that. We're here to understand what's actua
 
 Remember when we `cloned` the _Remote Repository_ a while ago? At that point it didn't only contain this tutorial and `Alice.txt` but actually two branches. 
 
-The _master_ we just went ahead and started working on, and one I called "tutorial_wip" on which I commit all the changes I make to this tutorial. 
+The _master_ we just went ahead and started working on, and the one I called "tutorial_wip" on which I commit all the changes I make to this tutorial. 
 
 When we copied the things in the _Remote Repository_ into your _Dev Environment_ a few extra steps happened under the hood. 
 
 Git setup the _remote_ of your _Local Repository_ to be the _Remote Repository_ you cloned and gave it the default name `origin`. 
 
 >Your _Local Repository_ can track several _remotes_ and they can have different names, but we'll stick to the `origin` and nothing else for this tutorial. 
 
-Then it copied the two remote branches into your _Local Repository_ and finally it `checked out` _master_ for you. 
+Then it copied the two remote branches into your _Local Repository_ and finally it `checkout`ed _master_ for you. 
```

Comment:
That works 😅 

I found it really hard to find a balance between referencing the exact commands and writing readable sentences